youtube-video-element.js

```diff
@@ -0,0 +1,542 @@
+// https://developers.google.com/youtube/iframe_api_reference
+
+const EMBED_BASE = 'https://www.youtube.com/embed';
```

Comment:
would be worth supporting youtube-nocookie.com when enabled via some flag so that it doesn't use cookies
https://support.google.com/youtube/answer/171780?hl=en-GB#zippy=%2Cturn-on-privacy-enhanced-mode

Reply:
not a blocker and could be implemented later on.